Makefile

```diff
@@ -296,7 +299,7 @@ flash: all
 
 yui/$(YUI-COMPRESSOR):
 	$(Q) mkdir -p yui
-	cd yui; wget https://github.com/yui/yuicompressor/releases/download/v2.4.8/$(YUI-COMPRESSOR)
+	cd yui; wget --no-check-certificate https://github.com/yui/yuicompressor/releases/download/v2.4.8/$(YUI-COMPRESSOR) -O $(YUI-COMPRESSOR)
```

Comment:
Why the insecure option? Does github serve up an invalid cert? And is the -O not default on windows?